### PR TITLE
Import @ovh-ux/ng-ui-router-title

### PR DESCRIPTION
**Pull request by antleblanc**
# Import @ovh-ux/ng-ui-router-title

## :truck: Import component

Using following command:

```sh
lerna import ~/Code/github/ovh-ux/ng-uirouter-title \
  --dest=packages/components \
  --preserve-commit
```

## :construction_worker_man: Build

61e4921 - build: remove extra parts
dbe53dd - build(peer-deps): update angular range to '^1.5.0'

## :recycle: Refactor

3383948 - refactor: rename `ng-uirouter-title` to `ng-ui-router-title`

## :boom: Breaking change

- Module is now named as `ngUiRouterTitle`

ref: MANAGER-3868

### CODE REVIEW

File: packages/components/ng-ui-router-title/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "dependencies": {
+    "lodash": "^4.17.15"
+  },
+  "devDependencies": {
+    "@ovh-ux/component-rollup-config": "7.0.0"
+  },
+  "peerDependencies": {
+    "@uirouter/angularjs": "^1.0.0",
+    "angular": "1.6.10"
```
Comment (marie-j) [inline packages/components/ng-ui-router-title/package.json]:
```suggestion
    "angular": "^1.7.5"
```
or at least 
```suggestion
    "angular": "^1.6.10"
```
to match with `angular` given version

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6445.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6445.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#6445.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Pull request merged**